### PR TITLE
#390 Move all members from the deleted team to the noTeamParticipants collection

### PR DESCRIPTION
**Pull request by nikolayninov**

### CODE REVIEW

File: packages/api/controllers/hackathon_team_controller.go [hunk 1]
```diff
@@ -202,6 +202,27 @@ func DeleteHackathonTeams(c *fiber.Ctx) error {
...
+
+	var tempTeamMember models.TeamMember
+
+	for i := 0; i < len(team.TeamMembers); i++ {
+		key_from_hex, _ := primitive.ObjectIDFromHex(team.TeamMembers[i])
+		_ = teamMembersCollection.FindOne(ctx, bson.M{"_id": key_from_hex}).Decode(&tempTeamMember)
+		teamMembers[i] = tempTeamMember
+	}
+
+	BatchAddTeamMembersToNoParticipants(ctx, teamMembers)
```
Comment (asynchroza) [inline packages/api/controllers/hackathon_team_controller.go:223]: Can you handle exceptions here?

**Review by asynchroza — APPROVED** [review]
Please, handle possible exceptions, otherwise it looks good ⚡️⚡️

File: packages/api/controllers/hackathon_team_controller.go (hunk 1 above)
Comment (nikolayninov) [reply packages/api/controllers/hackathon_team_controller.go:223]: Should it return 500 as status code or?

Comment (asynchroza) [reply packages/api/controllers/hackathon_team_controller.go:223]: yep

```go
_, err := BatchAddTeamMembersToNoParticipants(ctx, teamMembers)
if err != nil {
    return c.Status(blah, blah, blah, 500, message: "Failed moving members")
}
 
```

**Comment by asynchroza** [comment]
nini, add the hashtag with the number at the beginning of your commits msgs as well.

File: packages/api/controllers/hackathon_team_controller.go (hunk 1 above)
Comment (nikolayninov) [reply packages/api/controllers/hackathon_team_controller.go:223]: Yeah I did that, just needed the green light to commit

Comment (asynchroza) [reply packages/api/controllers/hackathon_team_controller.go]: don't be afraid to commit and push 😄. We can easily revert back if something is not ok. By pushing I get notified as a codeowner and I can check the code asap if all is good.

**Pull request merged**